__tests__/app.test.js

```diff
@@ -64,8 +64,52 @@ describe('GET /api/reviews', () => {
         .get('/api/review')
         .expect(404)
         .then((response) => {
-            const message = response.body.message
-            expect(message).toBe('SORRY NO SUCH PATH ;(')
+            const message = response.body.msg
+            expect(message).toBe('NOT FOUND!')
         })
     })
 });
+
+describe('GET /api/reviews/:review_id', () => {
+    test('status:200 returns an object with required properties', () => {
+        const ID = 1;
+        return request(app)
+        .get(`/api/reviews/${ID}`)
+        .expect(200)
+        .then(({body}) => {
+            expect(body.review).toEqual({
```

Comment:
This is fine but I'd probably use .toMatchObject here just incase the test data was to change in the future. Don't worry about changing it now though.